.github/workflows/test-action.yml

```diff
@@ -0,0 +1,28 @@
+name: Test generate-metadata-action
+
+on:
+  push:
+    branches:
+      - "*"
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  action-run:
+    runs-on: ubuntu-latest
+    outputs:
```

Comment:
I don't think this job level output needs to be set, as the action output can be accessed directly from the action step with `steps.generate_metadata.outputs.filepath` as below